### ISSUE TITLE
Questions Regarding Hyperparameters in the Code

### ISSUE THREAD

**Issue by jixiedy** [kqwang/DLPR#1]
Thank you very much for your outstanding work. I have a few questions regarding your open-source code:

1.  Many parameters in your code do not seem to be detailed in your paper’s experimental setup. Could you please confirm whether all the default settings in the code, including wavelength (632.8e-6), pixel size (8e-3), and prop_dis (20), are the optimal parameter settings used in your experiments?
2.  For the phase_min and phase_max parameters in dataset_generation.py, the default values in the code are both set to 1. Are these the optimal settings used in your experiments? If not, could you provide recommended settings for phase_min and phase_max?

**Comment by kqwang** [comment]
@jixiedy It's an honor to have your attention. The paper basically compares DD and PD simulately. In order to create a fair comparison environment, I set these hyperparameters to be uniform for DD and PD, such as wavelength (632.8e-6), pixel size (8e-3), and prop_dis (20). You can modify them accordingly to suit your experimental setup. Similarly, both phase_min and phase_max are uniformly set to 1 when generating simulation data for comparison of DD and PD. If the neural network trained on simulated data is to be used for testing on experimental data, phase_min and phase_max need to be modified accordingly to the appropriate range. The larger the range the more adaptable the neural network is to the phase range, and accordingly there may be a slight decrease in the network accuracy. In addition, I added a new hyperparameter "pad" to eliminate “edge diffraction effects”.
